controllers/openstackmachine_controller.go

```diff
@@ -52,32 +52,30 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-const (
-	InstanceIDIndex = ".spec.instanceID"
+// OpenStackMachineReconciler reconciles a OpenStackMachine object.
+type OpenStackMachineReconciler struct {
+	Client           client.Client
+	Log              logr.Logger
```

Comment:
I think `Log` can be removed here in favor of using the logger from the controller-runtime context.

Reply:
Yup, it was unused after your other suggested changes to SetupWithManager